src/dogstatsd_vm_stats.erl

```diff
@@ -124,8 +124,13 @@ handle_info({timeout, R, ?TIMER_MSG}, S = #state{key=K, delay=D, timer_ref=R}) -
     dogstatsd:gauge([K,"run_queue"], erlang:statistics(run_queue), 1.00),
 
     %% Error logger backlog (lower is better)
-    {_, MQL} = process_info(whereis(error_logger), message_queue_len),
-    dogstatsd:gauge([K,"error_logger_queue_len"], MQL, 1.00),
+    %% For Elixir 1.7/OTP 21+ error_logger is not started by default
+    case whereis(error_logger) of
+      undefined -> ok;
+      pid ->
```

Comment:
This looks great! I was thinking of caching this, but since the default is to generate stats every 60 seconds I think you are correct to avoid that premature optimization. :+1: